sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_delete_images_async.py

```diff
@@ -41,7 +48,7 @@ def __init__(self):
     async def delete_images(self):
         load_registry()
         # Instantiate an instance of ContainerRegistryClient
-        async with ContainerRegistryClient(self.endpoint, self.credential, audience=self.audience) as client:
+        async with ContainerRegistryClient(self.endpoint, self.credential, audience=self.audience) as client: # type: ignore[arg-type]
```

Comment:
What were these `ignore` statements in the samples for?

Reply:
I don't remember, I'll remove it and check the pipeline result.

Reply:
Interesting, the error is no longer there after remove one. I'll remove the rest.